custom/icds_reports/static/icds_reports/mobile/css/map-chart.less

```diff
@@ -118,12 +118,16 @@ a:active, a:hover, a:focus {
   font-size: 10px;
 }
 
-// reduce width of tooltip
-.nvtooltip .hoverinfo {
-  width: 160px !important;
+//wrapping text if it goes beyond the width of tooltip
+.nvtooltip {
+  width: 150px !important;
+  word-break: break-word;
+  white-space: normal;
+  padding: 5px;
 }
 
-// move tooltip 40px to the left to accommodate margins
-.nvtooltip {
-  margin-left: -40px;
+//by default labels and vertical lines opacity is made 0 instead of
+// wrapping when label is long to avoid overlap
+#mobile-bar-chart text, #mobile-bar-chart line {
+  opacity: 1 !important;
```

Comment:
what behavior does this change? do you have like a before/after?

Reply:
Before: when labels get closer, after a certain limit labels are hidden to avoid overlapping
![Screenshot from 2020-01-31 18-00-30](https://user-images.githubusercontent.com/56248457/73539648-e600ae00-4453-11ea-9bd4-bd60f4cf2c61.png)

after:
![Screenshot from 2020-01-31 18-01-49](https://user-images.githubusercontent.com/56248457/73539711-10526b80-4454-11ea-87ef-946c2a5ab14c.png)

Reply:
:+1: